apps/notifications/src/panel/templates/button-like.jsx

```diff
@@ -2,44 +2,54 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useContext } from 'react';
 import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import { setLikeStatus } from '../flux/note-actions';
+import { setLikeStatus } from '../state/notes/thunks/index';
 import ActionButton from './action-button';
 import { keys } from '../helpers/input';
 import { getReferenceId } from '../helpers/notes';
+import { RestClientContext } from '../Notifications';
 
-const LikeButton = ( { commentId, isLiked, note, translate } ) => (
-	<ActionButton
-		{ ...{
-			icon: 'star',
-			isActive: isLiked,
-			hotkey: keys.KEY_L,
-			onToggle: () =>
+const LikeButton = ( { commentId, isLiked, note, translate, setLikeStatus } ) => {
+	const restClient = useContext( RestClientContext );
+
+	return (
+		<ActionButton
+			icon={ 'star' }
```

Comment:
Nit: I'd alter that to:
```suggestion
			icon="star"
```